ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -891,6 +896,71 @@ - (Class)questionResultClass {
 
 @end
 
+
+#pragma mark - ORKEligibilityAnswerFormat
+
+@implementation ORKEligibilityAnswerFormat
+
+- (instancetype)init {
+    self = [self initWithPreferredAnswer:YES];
+    return self;
+}
+
+- (instancetype)initWithPreferredAnswer:(BOOL)preferredAnswer {
+    self = [super init];
+    if (self) {
+        _preferredAnswer = preferredAnswer;
+    }
+    return self;
+}
+
+- (ORKQuestionType) questionType {
```

Comment:
All the other answer formats' `questionType` follow the same syntax, so I thought to mimic it as well. I will go ahead and remove the space from all of them.